packages/core/src/actors/transition.ts

```diff
@@ -12,16 +12,17 @@ import { ActorLogic, ActorContext, ActorSystem, EventObject } from '../types';
 export function fromTransition<
   TState,
   TEvent extends EventObject,
-  TSystem extends ActorSystem<any>
+  TSystem extends ActorSystem<any>,
+  TInput
 >(
   transition: (
     state: TState,
     event: TEvent,
     actorContext: ActorContext<TEvent, TState, TSystem>
   ) => TState,
-  initialState: TState | (({ input }: { input: any }) => TState) // TODO: type
+  initialState: TState | (({ input }: { input: TInput }) => TState) // TODO: type
 ): ActorLogic<TEvent, TState, TState> {
-  const logic: ActorLogic<TEvent, TState, TState, TState> = {
+  const logic: ActorLogic<TEvent, TState, TState> = {
```

Comment:
TODO: Since `TInput` was introduced to this actor type - it should be passed to this `ActorLogic` here as well

Reply:
already fixed this